docs/components/table/README.md

```diff
@@ -382,12 +383,28 @@ will emit the `filtered` event, passing a single argument which is the complete
 items passing the filter routine. Treat this argument as read-only.
 
 ### Sorting
+As mentioned above in the **fields** section, you can make columns sortable. Clciking on
+sortable a column header will sort the column in ascending direction, while clicking
+on it again will switch the direction or sorting.  Clicking on a non-sortable column
```

Comment:
I removed a reset by clicking on non-sortable column, because it just set sortBy to `null`, but parameter sort=null|asc still sending to backend API.
On another tought, is may be up to Item provider func to deal with null sort target and remove it from query.
And on third thought, maybe better not expicitly to null `sortBy`, but reset it to default sortBy, defined at startup. If it not defined, then to null it.